src/index.css

```
@@ -1,50 +1,1071 @@
+}
+
+.months button,
+.view-btn,
+.btn,
+.expense-form button,
+.budget-row button,
+.home-actions a {
+  border-radius: 999px;
+  border: 1px solid var(--color-border);
+  background: var(--color-surface);
+  color: var(--color-primary);
```

Stylelint (17.9.0)

[error] Duplicate property "color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)